pkgs/development/compilers/dasm/default.nix

```
@@ -0,0 +1,33 @@
+{stdenv, fetchFromGitHub}:
+
+stdenv.mkDerivation rec {
+	pname = "dasm";
+	version = "2.20.13";
+
+	src = fetchFromGitHub {
+		owner = "dasm-assembler";
+		repo = "dasm";
+		rev = "2.20.13";
```

```suggestion
		rev = version;
```

I meant literally put the variable here. This way you don't need to change two variables when updating this `dasm`, only the first one.

---

Oh! Beautiful. Thanks, again!